Client/src/Remote/RemoteHelper.tsx

```diff
@@ -42,6 +48,13 @@ const fetchRemote = async (url: string, method: string, data: any = {}): Promise
     }
 
     const message = await response.json()
+
+    if (response.status.toString().charAt(0) == '2' && window.location.href.indexOf("/sign-up") > -1) {
```

Comment:
this shouldnt be handled here, i think you're getting something when you are calling post call('/sign-up') fro callSignUp(user) this file here, shouldnt contain any specific to one endpoint code, this file is a helper for generic stuff